pandas/tests/frame/test_arithmetic.py

```diff
@@ -374,13 +374,12 @@ def test_floordiv_axis0(self):
         result2 = df.floordiv(ser.values, axis=0)
         tm.assert_frame_equal(result2, expected)
 
-    @pytest.mark.slow
     @pytest.mark.parametrize("opname", ["floordiv", "pow"])
     def test_floordiv_axis0_numexpr_path(self, opname):
         # case that goes through numexpr and has to fall back to masked_arith_op
         op = getattr(operator, opname)
 
-        arr = np.arange(10 ** 6).reshape(100, -1)
+        arr = np.arange(10 ** 3).reshape(100, -1) * 1000
```

Comment:
going through the numexpr pth requires having at least 10k elements

Reply:
OK, I missed something here. I will have a look to numexpr. There is no room for improvemnt if 10k elements are needed.
Can you close is PR ?

Reply:
> There is no room for improvemnt if 10k elements are needed.

Could use 10k instead of 1e6?

> I will have a look to numexpr.

The relevant code is in pandas.core.computations.expressions.  Look for `_MIN_ELEMENTS`

> Can you close is PR ?

If you want to close it, there should be a button at the bottom of this page

Reply:
OK, thanks, I've proposed a new PR with 10**4 elements. It is indeed worth it.

I proposed to add an additional test to this one:
```
from pandas.core.computation.expressions import  _USE_NUMEXPR
assert _USE_NUMEXPR
```
Is it worth it ?